src/applications/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -120,20 +113,11 @@ const RatedDisabilityList = ({
 
   const serviceConnected = getServiceConnectedDisabilities(
     ratedDisabilities?.ratedDisabilities,
-  );
-
+  ).sort(sortFunc);
   const nonServiceConnected = getNonServiceConnectedDisabilities(
```

Comment:
Not sorting this one anymore because the only two keys that can be sorted by are effectiveDate and ratingPercentage; these values are both null for non-service connected ratings